src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto" , sans-serif;
+
+  --main-color: #00acdc;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.container {
+  max-width: 1200px;
+  min-width: 1024px;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.logo_link {
+  display: flex;
+  align-items: center;
+}
+
+.flex_container {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.flex_container .logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+}
+
+.nav__link {
+  padding: 23px;
```

Comment:
Change the padding value to 'padding: 23px 0;' to avoid unnecessary horizontal spacing between menu items.